scholly/base/src/components/publications/publicationsLayout/LayoutByCategory.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * The LayoutByCategory component displays a list of publications group by category type selcted by user
+ */
+ import React from 'react';
+ import { categoryType, categoryPageSize } from '../../../config/publications';
+ import LayoutAllPublications from './LayoutAllPublications';
+
+ const LayoutByCategory = ({ teamPublications }) => {
+  const renderPublicationsByCategory = (categoryType) => {
+    const publicationsByCategory = teamPublications.filter(
+      (pub) => pub.category.type === categoryType.toUpperCase()
+    );
+    return (
+      publicationsByCategory.length > 0 && (
+        <>
```

Comment:
Personally would prefer using `<Fragment>` instead of `<>`,  although the only benefit of [using `<Fragment>` is that it supports keys](https://reactjs.org/docs/fragments.html#:~:text=you%20can%20use%20%3C%3E%3C%2F%3E%20the%20same%20way%20you%E2%80%99d%20use%20any%20other%20element%20except%20that%20it%20doesn%E2%80%99t%20support%20keys%20or%20attributes.), and might [support other attributes like event handlers in the future](https://reactjs.org/docs/fragments.html#:~:text=In%20the,Live). But it is not a problem for now ig.

Reply:
My 2c from trying to follow the airbnb style guide in my branch: they enforce <> if the file is jsx, but I also went and changed all usages of Fragment to just <> in general. I personally think just leave it as <> for now.

Reply:
Don't mind me, agree with Vanessa and also following rules from [eslint](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md#:~:text=this%20is%20the%20default%20mode.%20it%20will%20enforce%20the%20shorthand%20syntax%20for%20react%20fragments%2C%20with%20one%20exception.%20keys%20or%20attributes%20are%20not%20supported%20by%20the%20shorthand%20syntax%2C%20so%20the%20rule%20will%20not%20warn%20on%20standard-form%20fragments%20that%20use%20those.)